### PR TITLE
Add state delta subscription via websockets

### PR DESCRIPTION
**Pull request by leebradley**
Add documentation to describe how to subscribe to state delta events by
using the scabbard /ws/subscribe endpoint.

Signed-off-by: Lee Bradley <bradley@bitwise.io>

### CODE REVIEW

File: docs/0.7/howto/state_delta_websockets.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# State Delta Subscriptions via Web Sockets
+
+<!--
+  Copyright 2022 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+As transactions are committed to the blockchain, an app developer may be
```
Comment (vaporos) [inline docs/0.7/howto/state_delta_websockets.md:9]: s/the blockchain/Scabbard/

File: docs/0.7/howto/state_delta_websockets.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+interested in receiving events related to the changes in state that
+result. These events, *StateDeltaEvents*, include
+information about the advance of the blockchain, as well as state
```
Comment (vaporos) [inline docs/0.7/howto/state_delta_websockets.md:12]: s/blockchain/distributed ledger/

File: docs/0.7/index.md [hunk 3]
```diff
@@ -252,6 +252,9 @@ to another database, event stream, log, or other mechanism. This allows
 applications to query and analyze the smart contract state in their own
 business-specific databases.
 
+To read state delta events in real time, use [State Delta Subscriptions via Web
+Sockets]({% link docs/0.7/howto/state_delta_websockets.md %}).
```
Comment (vaporos) [inline docs/0.7/index.md:256]: This should also be linked on the sidebar under the "For Developers" section.

File: docs/0.7/howto/state_delta_websockets.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+An application can subscribe to receive these events via a web socket,
+provided by the REST API component.
+
+# Opening a Web Socket
+
+The application developer must first open a web socket.
+
+Connect to the websocket at the following URL:
+```
+/{circuit_id}/{service_id}/ws/subscribe?last_seen_event={last_event_id}
```
Comment (peterschwarz) [inline docs/0.7/howto/state_delta_websockets.md:24]: This begins with `/scabbard`

**Review by peterschwarz — CHANGES_REQUESTED** [review]

File: docs/0.7/howto/state_delta_websockets.md (hunk 4 above)
Comment (leebradley) [reply docs/0.7/howto/state_delta_websockets.md]: Good catch. I even wrote out an app to test the ws subscribe action and somehow missed that part of the url. Fixed!

**Review by vaporos — APPROVED** [review]

**Review by peterschwarz — APPROVED** [review]

**Pull request merged**